lib/open_project/help_link/engine.rb

```diff
@@ -26,7 +26,7 @@ class Engine < ::Rails::Engine
     engine_name :openproject_help_link
 
     def self.settings
-      { :default => {"help_link_target" => "https://www.openproject.org/projects/support"},
+      { :default => {"help_link_target" => "https://www.openproject.org/help"},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.